log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java

```diff
@@ -290,6 +291,7 @@ public void activateOptions() {
         props.put(DELIVERY_TIMEOUT_MS_CONFIG, deliveryTimeoutMs);
         props.put(LINGER_MS_CONFIG, lingerMs);
         props.put(BATCH_SIZE_CONFIG, batchSize);
+        props.put(ENABLE_IDEMPOTENCE_CONFIG, false);
```

Comment:
@yyu1993 Can you please add a comment explaining this?

Reply:
I have added the comment.